po/ru.po

```diff
@@ -1878,7 +1878,7 @@ msgstr "&Информация"
 
 #: dnfdragora/ui.py:605
 msgid "&History"
-msgstr ""
+msgstr "История"
```

Comment:
Please keep accelerators in place. Somebody might want to use them.

Reply:
Yeah, that's for sure.